packages/cozy-webview/src/api/services/NativeService.ts

```diff
@@ -0,0 +1,31 @@
+import { ParentHandshake } from 'post-me'
+
+import { MessengerRegister } from '@api/models/messengers'
+import { NativeEvent } from '@api/models/events'
+import { NativeMessenger } from '@api/services/NativeMessenger'
+import { NativeMethods } from '@api/models/methods'
+import { WebviewRef } from '@api/models/environments'
+
+export class NativeService {
+  private messengerService = NativeMessenger
+
+  private messengerRegister = {} as MessengerRegister
+
+  public registerWebview = async (
+    webviewRef: WebviewRef,
+    localMethods?: NativeMethods
+  ): Promise<void> => {
+    const messenger = new this.messengerService(webviewRef)
+
+    this.messengerRegister = {
+      ...this.messengerRegister,
+      [webviewRef.props.source.uri]: { messenger }
+    }
+
+    await ParentHandshake(messenger, localMethods, 50, 100)
```

Comment:
Can we extract those numbers into named constants so the reader can understand what they mean without having to read the post-me doc?